core/src/types.rs

```diff
@@ -121,7 +121,7 @@ impl<T: Clone> Cloned<T> for Box<T> where T: Sized + Clone {
 /// This type must implement `FromRc` and `Cloned`, and a `From` implementation must exist for
 /// `TypedValue`.
 ///
-pub type ValueRc<T> = Rc<T>;
+pub type ValueRc<T> = Arc<T>;
```

Comment:
Can we make this its own tiny PR?

Reply:
(And comment about it in #656)

Reply:
https://github.com/mozilla/mentat/pull/678

Reply:
Needs a rebase on `master`…